cirq-web/cirq_ts/dist/index.html

```diff
@@ -0,0 +1,14 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <title>Cirq Web Development page</title>
+</head>
+<body>
+  <div id="container"></div>
+  <script src="bloch_sphere.bundle.js"></script>
+  <script>
+    const blochSphere = renderBlochSphere('{"radius": 5, "hMeridians": 1, "vMeridians": 1}');
+    blochSphere.addVector();
```

Comment:
Recommend passing in a `VectorData` value here, so the reader can see what the format looks like. This could also be presented as an opportunity for users to try out hot-reloading, e.g. by flipping the vector from |0) to |1).

Reply:
Done